frontend/src/app/modules/auth/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="sign-up">
+    <form name="authForm" #authForm="ngForm" class="registration-form">
+        <h1>Sign In</h1>
+        <label>
+            <mat-error *ngIf="submitted">{{ getEmailErrorMessage() }}</mat-error>
+            <input
+                type="text"
+                [(ngModel)]="email"
+                matInput
+                required
+                placeholder="Email"
+                [ngModelOptions]="{ standalone: true }" />
+        </label>
+        <label>
+            <mat-error *ngIf="submitted">{{ getPasswordErrorMessage() }}</mat-error>
+            <input
+                type="password"
+                [(ngModel)]="password"
+                #upPassword="ngModel"
+                required
+                placeholder="Password"
+                [ngModelOptions]="{ standalone: true }" />
+        </label>
+        <button mat-button (click)="validation()">Confirm</button>
+    </form>
+    <div class="image-section">
+        <div class="car-background"></div>
+        <img class="car" src="assets/topic-icons/Racing Car.svg" />
+
+        <div class="camera-background"></div>
+        <img class="camera" src="assets/topic-icons/Camera.svg" />
+
+        <div class="clover-background"></div>
+        <img class="clover" src="assets/topic-icons/Four Leaf Clover.svg" />
+
+        <div class="cook-background"></div>
+        <img class="cook" src="assets/topic-icons/Cook.svg" />
+
+        <div class="dancing-background"></div>
+        <img class="dancing" src="assets/topic-icons/Woman Dancing.svg" />
+
+        <div class="dancing-background"></div>
+        <img class="dancing" src="assets/topic-icons/Woman Dancing.svg" />
+
+        <div class="dog-background"></div>
+        <img class="dog" src="assets/topic-icons/Dog Face.svg" />
+
+        <div class="monocle-face-background"></div>
+        <img class="monocle-face" src="assets/topic-icons/Face With Monocle.svg" />
+        <h1>
+            Learn English
+            <div>With Logo</div>
+        </h1>
```

Comment:
Can you move it to separate component? Not to repeat it)

Reply:
fixed